frontend/src/components/layout/Footer/Footer.tsx

```diff
@@ -1,19 +1,60 @@
-import './Footer.scss';
-
 import React from 'react';
-import { Navbar, Nav } from 'react-bootstrap';
+import { ApiVersionInfo } from 'components/common/ApiVersionInfo';
+import styled from 'styled-components';
 
+/**
+ * Display common links and the api version in the bottom footer.
+ * @returns Footer component.
+ */
 function Footer() {
   return (
-    <Navbar expand className="footer">
-      <Nav.Link href="http://www.gov.bc.ca/gov/content/home/disclaimer">Disclaimer</Nav.Link>
-      <Nav.Link href="http://www.gov.bc.ca/gov/content/home/privacy">Privacy</Nav.Link>
-      <Nav.Link href="http://www.gov.bc.ca/gov/content/home/accessible-government">
-        Accessibility
-      </Nav.Link>
-      <Nav.Link href="http://www.gov.bc.ca/gov/content/home/copyright">Copyright</Nav.Link>
-    </Navbar>
+    <FooterStyled>
+      <a href="http://www.gov.bc.ca/gov/content/home/disclaimer">Disclaimer</a>
+      <a href="http://www.gov.bc.ca/gov/content/home/privacy">Privacy</a>
+      <a href="http://www.gov.bc.ca/gov/content/home/accessible-government">Accessibility</a>
+      <a href="http://www.gov.bc.ca/gov/content/home/copyright">Copyright</a>
+      <ApiVersionInfo />
+    </FooterStyled>
   );
 }
 
+/**
+ * Styled footer component.
+ */
+const FooterStyled = styled('div')`
```

Comment:
I think eventually we want to try and promote reuse a bit more by putting splitting of some of these components and placing them in a shared location (ie, the a portion). Not necessarily required here however.